src/NovelLoggingService.cpp

```diff
@@ -26,8 +26,7 @@ NovelLoggingService::NovelLoggingService() {
      spdlog::set_error_handler([](const std::string& msg) {
         std::cerr << "SPDLOG ERROR: " << msg << std::endl;
     });
-  }
-  catch (const spdlog::spdlog_ex &ex)
+  } catch (const spdlog::spdlog_ex &ex)
```

Comment:
@RubyNova does this look better?

Reply:
No - the problem is with the format of your reference symbol. It's on the rhs rather than lhs. Plzfix.

Reply:
Will be fixed in next push :+1: promise